src/class-methods/class-private-async-generator-method-static-whitespace-syntax-error.case

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2018 Bloomberg LP. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+ /*---
+desc: Syntax error if whitespace after '#' sigil
```

Comment:
I'm afraid this description might mess up with Yaml parsers used by different test runners. Following the convention from the quoted file should be fine:

`No space allowed between sigil and IdentifierName`

Reply:
You may also want to describe the file and text:

- `grammar-privatename-error-initializer.case`: `No space allowed between sigil and IdentifierName (class field with initializer)`
- `grammar-privatename-error-static-async-gen.case`: `No space allowed between sigil and IdentifierName (static async generator)`
- `grammar-privatename-error-async-gen.case`: `No space allowed between sigil and IdentifierName (async generator)`
- `grammar-privatename-error-static-gen.case`: `No space allowed between sigil and IdentifierName (static generator)`
- `grammar-privatename-error-gen.case`: `No space allowed between sigil and IdentifierName (generator)`
- `grammar-privatename-error-static-async.case`: `No space allowed between sigil and IdentifierName (static async method)`
- `grammar-privatename-error-async.case`: `No space allowed between sigil and IdentifierName (async method)`
- `grammar-privatename-error-static-meth.case`: `No space allowed between sigil and IdentifierName (static method)`
- `grammar-privatename-error-meth.case`: `No space allowed between sigil and IdentifierName (method)`
...